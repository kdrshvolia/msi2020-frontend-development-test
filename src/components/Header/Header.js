import React from 'react';
import PropTypes from 'prop-types';

const Header = ({ title, subtitle }) => {
  return (
    <div>
      <h1>{title}</h1>
      <h2>{subtitle}</h2>
    </div>
  );
};

Header.propTypes = {
  title: PropTypes.string.isRequired,
  subtitle: PropTypes.string.isRequired,
};

export default Header;
