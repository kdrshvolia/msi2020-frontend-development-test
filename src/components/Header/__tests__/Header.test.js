import React from 'react';import renderer from 'react-test-renderer';import Header from '../Header';describe('SvgIcon', () => {  it('renders correctly', () => {    const rendered = renderer.create(<Header title="title" subtitle="subtitle" />);    expect(rendered.toJSON()).toMatchSnapshot();  });});