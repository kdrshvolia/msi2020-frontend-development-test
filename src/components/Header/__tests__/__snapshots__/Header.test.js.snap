// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SvgIcon renders correctly 1`] = `
<div>
  <h1>
    title
  </h1>
  <h2>
    subtitle
  </h2>
</div>
`;
